protoc_plugin/Makefile

```diff
@@ -81,16 +81,7 @@ $(TEST_PROTO_LIBS): $(PLUGIN_SRC) $(TEST_PROTO_SRCS)
 	mkdir -p $(TEST_PROTO_DIR)
 
 	protoc\
-		--dart_out="disable_constructor_args:$(TEST_PROTO_DIR)"\
```

Comment:
So we no longer `disable_constror_args`? Is it not necessary?

Maybe we could rely on g3 tests for this. In g3 `disable_constructor_args` is the default as it causes binary size bloat.

Reply:
We're generating two versions of the test protos output for this one option - I'm just trying to dial things back a bit (we don't do this for other options). The disable_constructor_args option is tested in protoc_plugin/test/file_generator_test.dart and a bit in message_generator_test.dart.

Given the history in the changelog It sounds like having ctor args is the default externally, and not having them is the default internally.